modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,119 +1,54 @@
-====
+You can enable multipathing at installation time for nodes that were provisioned in {product-title} 4.8 or later. While post-installation support is available by activating multipathing via the machine config, enabling multipathing during installation is recommended.
+
+In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time.
 
 [IMPORTANT]
 ====
-On IBM Z and LinuxONE, you can enable multipathing only if you configured your cluster for it during installation. For more information, see "Creating {op-system-first} machines" in _Installing a cluster with z/VM on IBM Z and LinuxONE_.
+On IBM Z and LinuxONE, you can enable multipathing only if you configured your cluster for it during installation. For more information, see "Installing {op-system} and starting the {product-title} bootstrap process" in _Installing a cluster with z/VM on IBM Z and LinuxONE_.
 ====
 // Add xref once it's allowed.
 
 .Prerequisites
-* You have a running {product-title} cluster that uses version 4.7 or later.
+* You have a running {product-title} cluster that uses version 4.8 or later.
 * You are logged in to the cluster as a user with administrative privileges.
 
 .Procedure
 
-. To enable multipathing on control plane nodes (also known as the master nodes):
-
-* Create a machine config file, such as `99-master-kargs-mpath.yaml`, that instructs the cluster to add the `master` label and that identifies the multipath kernel argument, for example:
-
+* To enable the multipath service at installation time:
 +
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: "master"
-  name: 99-master-kargs-mpath
-spec:
-  kernelArguments:
-    - rd.multipath=default
-    - root=/dev/disk/by-label/dm-mpath-root
-----
-
-. To enable multipathing on worker nodes:
-
-* Create a machine config file, such as `99-worker-kargs-mpath.yaml`, that instructs the cluster to add the `worker` label and that identifies the multipath kernel argument, for example:
+. Turn on multipath in the installation environment.
 +
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: "worker"
-  name: 99-worker-kargs-mpath
-spec:
-  kernelArguments:
-    - rd.multipath=default
-    - root=/dev/disk/by-label/dm-mpath-root
-----
-
-. Create the new machine config by using either the master or worker YAML file you previously created:
+** For a PXE or ISO boot, add the `rd.multipath=default` kernel argument to your `MachineConfig` object file. For information about setting kernel arguments, see "Adding kernel arguments to nodes".
 +
-[source,terminal]
-----
-$ oc create -f ./99-master-kargs-mpath.yaml
-----
-
-. Check the machine configs to see that the new one was added:
+** For a non-PXE/ISO boot, run the following commands:
```

Comment:
How about something like:

- Turn on multipath using `mpathconf --enable && systemctl start multipathd.service` (for more information, see the [RHEL 8 multipath documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_device_mapper_multipath/assembly_setting-up-dm-multipath-configuring-device-mapper-multipath#proc_basic-multipath-setup-setting-up-dm-multipath)). If booting the PXE or ISO, you may also instead add `rd.multipath=default` to turn on multipath from the kernel command-line.
- This will allow you to target the multipath device directly for installation using e.g. `/dev/mapper/mpatha` or a WWN-named symlink in `/dev/disk/by-id`. As usual, the device path can be specified on the kernel command-line (i.e. via `coreos.inst.install_dev`) or interactively (i.e. when invoking `coreos-installer`).

? Not sure if it's too wordy.

Reply:
@jlebon Made some more tweaks by incorporating the RHEL 8 docs. Thanks for the link. I need to make it task-based, so that's why it's broken into what seems like so many steps. Hoping I got the ordering and context right but please let me know. 🙇